tests/stat_tests/test_missing_values_autodist.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import pandas as pd
+import numbers
+
+import pytest
+
+from probatus.stat_tests import ks, psi, DistributionStatistics, AutoDist
+from sklearn.datasets import make_classification
+
+def test_missing_values_in_autodist():
+    """Test missing values have no impact in AutoDist functionality. """
+    # Create dummy dataframe
+    X, y = make_classification(10000, 10)
+    X = pd.DataFrame(X)
+    # Split train and test
+    X_train = X.sample(8000)
```

Comment:
Let's use train test split from sklearn. Using sample function you will get an overlap between the two.

Reply:
Done